security/compliance_operator/compliance-operator-installation.adoc

```diff
@@ -0,0 +1,140 @@
+[id="compliance-operator-installation"]
+= Installing the Compliance Operator
+include::modules/common-attributes.adoc[]
+:context: compliance-operator-installation
+
+toc::[]
+
+
+Before you can use the Operator, you must ensure it is deployed in the cluster.
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+. Create the target namespace:
++
+[source,terminal]
+----
+$ oc create -f deploy/ns.yaml
+----
+
+. Install the Operator from the web console or from the CLI:
++
+[source,terminal]
+----
+$ oc create -f deploy/olm-catalog/operator-group.yaml
+----
++
+[source,terminal]
+----
+$ oc create -f deploy/olm-catalog/subscription.yaml
```

Comment:
```yaml
---
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: compliance-operator
  namespace: openshift-compliance
spec:
  installPlanApproval: Automatic
  name: compliance-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace
```

Reply:
Note that this subscription points to the operator coming from the official redhat operators source. This is what we support.